src/test/java/com/revature/services/test/QuestionServiceTest.java

```diff
@@ -84,35 +100,87 @@ public void getAllQuestionsByStatus() throws Exception {
 	/** @author ken */
 	@Test
 	public void updateQuestionAcceptedAnswerId() throws Exception {
-		Question question = new Question(1, 0, "Title", "Content", LocalDate.MIN, null, false, 1);
+		Question question = new Question(1, 0, "Title", "Content", ts, null, false, 1);
 	
-		//Question pageResult = new PageImpl<Question>(questions);
 		when(questionRepository.save(Mockito.any(Question.class)))
 		.thenReturn(question);
 		Question result = questionService.updateQuestionAcceptedAnswerId(question);
-		//assertThat(result).contains(question);
 		assertEquals(result, question);
 	}
 	
 	/** @author ken */
 	@Test
-	public void getAllQuestionsByID() throws Exception {
-		Question question = new Question(1, 0, "Title", "Content", LocalDate.MIN, null, false, 1);
+	public void getQuestionByID() throws Exception {
+		Question question = new Question(1, 0, "Title", "Content", ts, null, false, 1);
 
 		when(questionRepository.findById( Mockito.anyInt()))
 		.thenReturn(Optional.of(question));
 		Question result = questionService.findById(1);
+		assertEquals(result, question);
 	}
 	
 	/** @author James */
 	@Test
 	public void updateQuestion() throws Exception {
-		Question question = new Question(1, 0, "Title", "Content", LocalDate.MIN, null, false, 1);
+		Question question = new Question(1, 0, "Title", "Content", ts, null, false, 1);
 	
 		when(questionRepository.save(Mockito.any(Question.class)))
 		.thenReturn(question);
 		Question result = questionService.save(question);
 		assertEquals(result, question);
 	}
+	/**@author Bukadiri Trawally*/
+	@Test
+	public void updateQuestionAcceptedAnswerId_will_return_question() {
+		Question q = new Question(1,1,"title","content", ts, ts, false, 1);
+		Question q1 = new Question(1,1,"title","content", ts, ts, true, 1);
```

Comment:
We should not be using `java.sql.Date` unless you are conducting integration tests with the Database that necessitate using it. Use LocalDate for all internal application date handling.

Reply:
fixed